zombienet_tests/misc/0002-download-polkadot-from-pr.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+TEMP_DIR=/tmp/zombie
+
+# add /tmp/zombie as first `looking dir` to allow to overrides commands.
+mkdir -p $TEMP_DIR
+export PATH=$TEMP_DIR:$PATH
+
+cd $TEMP_DIR
+#curl -L -O https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/1810914/artifacts/file/artifacts/polkadot
```

Comment:
Nit: remove this, or make it explicit that it is for native/non ci testing

Reply:
Thanks, added the explicit note.